File: Python/1042.py
x = input().split()
a, b, c = x
a = int(a)
b = int(b)
c = int(c)


if a > b and a > c:
    d = a
    if b > c:
        e = b
        f = c
    else:
        e = c
        f = b
if b > a and b > c:
    d = b
    if a > c:
        e = a
        f = c
    else:
        e = c
        f = a
if c > a and c > b:
    d = c
    if a > b:
        e = a
        f = b
    else:
        f = a
        e = b
print(f)
print(e)
print(d)
print()
print(a)
print(b)
print(c)

File: C#/1042.cs
using System;
using System.Linq;



class Program {
    static void Main() 
    {
        int[] x = Console.ReadLine().Split().Select(int.Parse).ToArray();
        int[] y = new int[3];

        Array.Copy(x, y, 3);
        Array.Sort(x);
        foreach (var item in x)
        {
            Console.WriteLine(item);
        }
        Console.WriteLine();
        foreach (var item in y)
        {
            Console.WriteLine(item);
        }
    }
}
       